src/lib/logger/logger.ts

```diff
@@ -120,18 +111,28 @@ export function setup(options: LoggerConfig | LoggerConfigItem = [DEFAULT_LOGGER
     );
   }
   const pinoConfig = { level: loggerConfig.level };
+  let colors = 'colors' in loggerConfig ? loggerConfig.colors : process.stdout.isTTY;
+  if ('EXPERIMENTAL__VERDACCIO_LOGGER_COLORS' in process.env) {
+    colors = Boolean(process.env.EXPERIMENTAL__VERDACCIO_LOGGER_COLORS);
```

Comment:
I'm not sure what might be on the `EXPERIMENTAL__VERDACCIO_LOGGER_COLORS` variable, but I'd advise to have a more comprehensive approach, in some enviroment is much easier to modify the value rather remove it completely, so leet's use `==='true'` instead to be more explecit.
Otherwise JS :) is not happy with any input here.
![Screenshot from 2022-02-25 21-40-53](https://user-images.githubusercontent.com/558752/155799804-7a0a7650-ca3b-42ee-938e-ae92fa0385d8.png)

Reply:
OMG! you're right 🤦 😵‍💫 🤦 🤦